scripts/modules/b2b-account/users.js

```diff
@@ -301,7 +302,9 @@ define(["modules/mozu-utilities", "modules/jquery-mozu", 'modules/api', "undersc
             else {
                 collection = new UsersGridCollectionModel({});
             }
-           
+            if (viewB2BAccount) {
+                collection.filterBy("customerAccountId eq " + self.model.attributes.accountToView);
```

Comment:
This won't work because the Get B2B Account request doesn't accept any filters. You are requesting a single B2B account with the id defined on `require.mozuData('user').accountId` which will be the value on the user claims. Not the value you want here.

Example request:
`/api/commerce/customer/b2baccounts/1032/users?startIndex=0&sortBy=createDate%20asc&pageSize=5&filter=customerAccountId%20eq%201044`
Notice how 1032 is the user's associated account ID but 1044 is the account we're viewing. The filter doesn't actually work here since that API supports no filters.

You need to update the `B2BUsersGridCollectionModel` that is used for this grid to use the right ID.